install/kubernetes/helm/istio/values.yaml

```diff
@@ -31,6 +31,21 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
+    # based on measurements 100ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.
+    averageLatencyThreshold: 100ms
+    # disabled, logonly or enforce
+    loadsheddingMode: enforce
```

Comment:
do we need to expose these 2 params in top values.yaml?  seems overkilling.

Reply:
one 1 param at top level now.